route/index.ts

```diff
@@ -1,12 +1,22 @@
 import * as express from 'express';
 import { Request, Response } from 'express';
-import { profileIndex } from '../controller/profileController';
+import { favoriteUpdate, getFavoriteById, favoritesGet, favoriteStoreValidate, favoriteUpdateValidate, favoriteStore, favoriteDelete } from '../controller/favoriteController';
+import { cultureGet } from '../controller/cultureController';
+
+import { venuesGet, venuesGetDetails } from '../controller/venueController';
 
 // Route Declare
 const route = express.Router();
 
 // Route List
-route.get('/', profileIndex);
+route.get('/', venuesGet);
+route.get('/culture', cultureGet);
+route.get('/favorites', favoritesGet);
+route.get('/:venueId', venuesGetDetails);
+route.post('/favorites', favoriteStoreValidate, favoriteStore);
+route.get('/favorites/:id', getFavoriteById);
+route.put('/favorites/:id', favoriteUpdateValidate, favoriteUpdate);
+route.delete('/favorites/:id', favoriteDelete);
```

Comment:
Error: Cannot read property 'id' of undefined

Reply:
link itu bisa semua kok ditempatku.
coba cek tabel DEV_UserProfile udah ada isinya belum? sesuai akunmu kalo belum ada isinya error kyak gitu

Reply:
@hapehatelo aq ada share env di group, coba update dulu env nya, di tempat ku masih error